service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/AbstractDocmosisOrder.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.util.stream.Collectors.groupingBy;
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+
+@Getter
+@AllArgsConstructor
+public class AbstractDocmosisOrder extends AbstractDocmosisData {
+    private final DocmosisJudgeAndLegalAdvisor judgeAndLegalAdvisor;
+    private final String courtName;
+    private final String familyManCaseNumber;
+    private final String dateOfIssue;
+    private final String complianceDeadline;
+    private final List<DocmosisRespondent> respondents;
+    private final List<DocmosisChild> children;
```

Comment:
Probably not ideal with duplicating the Docmosis from the package name, but I can imagine it makes development easier due to naming clash, so it's fine.

Reply:
Yes, so I don't really want Docmosis in the name at all, I just think it's confusing to be using two different Child types in the same lines of code. Any other ideas for this? Maybe being the same name is fine?

Reply:
It's fine, but Java will force you to use fully typed name for one of them so you will have:
uk.gov.hmcts.reform.fpl.model.docmosis.Child which is a bit annoying as well. So you can import only one with clashing name and need to fully reference the other. 

I personally prefer fully referencing the less used one as it is anyway only one or two lines where the mapping happens, so maybe something you could consider.